desktop/core/src/desktop/js/apps/jobBrowser/components/impalaQueries/query-details/QueryDetails.test.ts

```diff
@@ -0,0 +1,41 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { shallowMount } from '@vue/test-utils';
+import QueryDetails from './QueryDetails.vue';
+
+import queryMockResponse from './__test_data__/impala_query_1.json';
+
+describe('QueryDetails.vue', () => {
+  it('should render', () => {
+    const wrapper = shallowMount(QueryDetails, {
+      propsData: {
+        query: {
+          ...queryMockResponse.query,
+          profile: queryMockResponse.profile
+        }
+      },
+      global: {
+        provide: {
+          showQueries: () => {
+            // Empty
+          }
+        }
+      }
+    });
+    expect(wrapper.element).toMatchSnapshot();
```

Comment:
Why are we only testing snapshots? What about interactions like click etc?
This goes for all the new components in this PR.

Reply:
Considering that many things would change including the UX, wouldn't it be better to add those once we revamp the UX and move to React.